source/code/plugins/in_npmd_server.rb

```diff
@@ -64,7 +70,7 @@ def configure(conf)
         RETRY_START_WAIT_TIME_SECS      = 5   # 5 seconds
         RETRY_START_ATTEMPTS_PER_BATCH  = 5
 
-        CMD_ENUMERATE_AGENT_PROCESSES = "ps aux | grep "
+        CMD_ENUMERATE_PROCESSES_PREFIX = "ps aux | grep "
```

Comment:
Requested change by @robbiezhang  in https://github.com/Microsoft/PowerShell-DSC-for-Linux/pull/187